os/arch/arm/src/amebasmart/amebasmart_idle.c

```diff
@@ -226,11 +175,7 @@ void up_idle(void)
 	 */
 
 	nxsched_process_timer();
-#else
-
-	/* Sleep until an interrupt occurs to save power */
 
-	up_idlepm();
 #endif
```

Comment:
Please add up_pm_core_sleep() here. Same rules should apply to all other chipsets as well. Please check.

Reply:
@edwakuwaku Since we are not calling up_idle() inside idle thread for pm enable case, then why we need up_pm_core_sleep() inside up_idle() ?

Reply:
In case without PM enabled, we should apply WFE/WFI in idle thread as well.
For API naming, maybe we can change to `up_core_low_power()`

Reply:
This doesn't seem to be a consensus because we've discussed it yet.

Is it necessary to call WFE/WFI even though CONFIG_PM is disabled?
In my opinion, we don't need to apply WFE when CONFIG_PM is disabled.